examples/data-objects/focus-tracker/package.json

```diff
@@ -39,7 +39,7 @@
   "dependencies": {
     "@fluid-experimental/data-objects": "^1.1.0",
     "@fluidframework/common-definitions": "^0.20.1",
-    "@fluidframework/common-utils": "^0.32.1",
+    "@fluidframework/common-utils": "^0.33.1000-0",
```

Comment:
@ssimic2 and @tylerbutler do we consider this a breaking change?

Reply:
Just bumping dependency may not be a breaking change. @tylerbutler ? 

@tylerbutler this is interesting from main-to-next integration. Would this commit go through without merge conflict ?  "next" should probably not be consuming "0.33.1000-0".

Reply:
If this were a definitions package I would think almost definitely yes should go in `next`.  But this is just a utils package.  Hopefully no API surface is returning a type defined in here - i.e. this thing should be all implementation details and be suitable for a minor release.  But maybe I'm over-simplifying it, or maybe we just want to play it safe :)